### ISSUE TITLE
How to resume training CT-CLIP

### ISSUE THREAD

**Issue by qic999** [ibrahimethemhamamci/CT-CLIP#12]
Thank you for sharing the excellent work. How can one resume from a pre-training stage and continue training?

**Comment by sezginerr** [comment]
Hello @qic999, 

You can add the following line after you define clip in `run_train.py`

```python
clip = clip.load("path_to_pretrained_model") 
```
This should continue pre-training stage. There is no scheduler in the zero-shot training so no need to load the optimizer state.
